include/swift/Runtime/Metadata.h

```diff
@@ -919,7 +919,7 @@ static const uintptr_t ObjCReservedBitsMask =
 static const unsigned ObjCReservedLowBits =
   SWIFT_ABI_X86_64_OBJC_NUM_RESERVED_LOW_BITS;
 
-#elif defined(__arm64__)
+#elif defined(__arm64__) || defined(__powerpc64__)
```

Comment:
The comment above this block says that `lib/IRGen/SwiftTargetInfo.cpp` needs adjustments (you need to add a `configurePowerPC` function there).